### ISSUE TITLE
Fix coverage reports

### ISSUE THREAD

**Issue by gavv** [roc-streaming/roc-go#35]
Coverage became 0% after merging #32.

**Comment by Asalle** [comment]
seems like the package `goveralls` needs to be installed in order to use coveralls with github actions

sources:
https://docs.coveralls.io/go
https://github.com/mattn/goveralls
https://github.com/marketplace/actions/actions-goveralls

**Comment by gavv** [comment]
Good catch! IIRC I copied gh actions config from another project which use goveralls indeed.
